eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+  Toolchain: "rush"
+  PackagePath: "" # For NPM toolchain only
+  PreIntegrationSteps: ""
+  EnvVars: []
+  MaxParallel: 0
+  Matrix:
+    Linux_Node10X:
+      OSVmImage: "ubuntu-16.04"
+      TestType: "node"
+    Windows_Node10X:
+      OSVmImage: "vs2017-win2016"
+      TestType: "node"
+    Mac_Node10X:
+      OSVmImage: "macOS-10.13"
+      TestType: "node"
+    Browser_Linux_Node10X:
+      OSVmImage: "ubuntu-16.04"
+      TestType: "browser"
+    Browser_Windows_Node10X:
+      OSVmImage: "vs2017-win2016"
+      TestType: "browser"
+    Browser_Mac_Node10X:
+      OSVmImage: "macOS-10.13"
+      TestType: "browser"
+
+jobs:
+  - job: "Integration"
+
+    dependsOn:
+      - Prepare
+
+    variables:
+      - template: ../variables/globals.yml
+      - name: GeneratedPackageTargets
+        value: $[dependencies.Prepare.outputs['SelectPackages.GeneratedPackageTargets']]
+
+    strategy:
+      maxParallel: ${{ parameters.MaxParallel }}
+      matrix:
+        ${{ each matrixEntry in parameters.Matrix }}:
+          ${{ matrixEntry.key }}:
+            # OSName:
+            ${{ if startsWith(matrixEntry.value.OSVmImage, 'ubuntu') }}:
```

Comment:
These better criteria belong in `archetype-sdk-tests.yml` too. It's an improvement over that v1.